### PR TITLE
Add release notes for 2.2.0.0 release

### PR DESCRIPTION
**Pull request by cwperks**
Signed-off-by: Craig Perkins <cwperx@amazon.com>

### Description

Add release notes for 2.2.0.0 release

### Category [Enhancement, New feature, Bug fix, Test fix, Refactoring, Maintenance, Documentation]

Maintenance

### Issues Resolved

https://github.com/opensearch-project/security-dashboards-plugin/issues/1042

### Check List
- [ ] New functionality includes testing
- [ ] New functionality has been documented
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: release-notes/opensearch-security-dashboards-plugin.release-notes-2.2.0.0.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Compatible with Opensearch-Dashboards 2.2.0
+
+### Test Fix
+
+* Add tests for account-nav-button when multitenancy is disabled ([#1020](https://github.com/opensearch-project/security-dashboards-plugin/pull/1020))
+
+### Bug Fix
+
+* Revert "Replace _opendistro route with _plugins ([#895](https://github.com/opensearch-project/security-dashboards-plugin/pull/895))" ([#1035](https://github.com/opensearch-project/security-dashboards-plugin/pull/1035))
```
Comment (peternied) [inline release-notes/opensearch-security-dashboards-plugin.release-notes-2.2.0.0.md:11]: Could we rewrite this so its more human readable?

**Review by peternied — CHANGES_REQUESTED** [review]

File: release-notes/opensearch-security-dashboards-plugin.release-notes-2.2.0.0.md (hunk 1 above)
Comment (cwperks) [reply release-notes/opensearch-security-dashboards-plugin.release-notes-2.2.0.0.md]: Changed this to "Fix bug in SAML support after renaming". Should we edit the name of the PR as well?

Comment (peternied) [reply release-notes/opensearch-security-dashboards-plugin.release-notes-2.2.0.0.md]: Users/admins read release notes, developers read PRs, its ok to leave it as it was IMO

**Review by peternied — DISMISSED** [review]

**Review by peternied — APPROVED** [review]
Thanks for getting a head start on the release notes

**Review by DarshitChanpura — APPROVED** [review]

**Pull request merged**